Formula/postgresql@13.rb

```diff
@@ -0,0 +1,191 @@
+class PostgresqlAT13 < Formula
+  desc "Object-relational database system"
+  homepage "https://www.postgresql.org/"
+  url "https://ftp.postgresql.org/pub/source/v13.4/postgresql-13.4.tar.bz2"
+  sha256 "ea93e10390245f1ce461a54eb5f99a48d8cabd3a08ce4d652ec2169a357bc0cd"
+  license "PostgreSQL"
+
+  livecheck do
+    url "https://ftp.postgresql.org/pub/source/"
+    regex(%r{href=["']?v?(13(?:\.\d+)+)/?["' >]}i)
+  end
+
+  keg_only :versioned_formula
+
+  # https://www.postgresql.org/support/versioning/
+  deprecate! date: "2025-11-13", because: :unsupported
```

Comment:
This should actually be disabled on that date. Maybe we deprecate it a year before?